user-interface/src/data-verification/transfer/CaseTable.tsx

```diff
@@ -40,15 +40,34 @@ function _CaseTable(props: CaseTableProps, CaseTableRef: React.Ref<CaseTableImpe
         <tr>
           {onSelect && <th scope="col">Select</th>}
           <th scope="col">Case Number</th>
+          <th scope="col">Court (Division)</th>
           <th scope="col">Case Title</th>
-          <th scope="col">SSN/EIN</th>
-          <th scope="col">Court</th>
-          <th scope="col">Case Filed</th>
           <th scope="col">Chapter</th>
+          <th scope="col">Filed Date</th>
+          <th scope="col">Tax ID</th>
         </tr>
       </thead>
       <tbody>
         {cases?.map((bCase, idx) => {
+          if (!bCase) {
+            if (!onSelect) return <></>;
+            return (
+              <tr key={'empty'} data-testid={'empty-row'}>
+                <th scope="col">
+                  <input
+                    type="radio"
+                    onChange={handleCaseSelection}
+                    value={idx}
+                    name="case-selection"
+                    data-testid={`suggested-cases-radio-empty`}
+                    checked={idx === selectedIdx}
+                    title={`case not listed`}
+                  ></input>
+                </th>
+                <td colSpan={6}>Case not listed.</td>
```

Comment:
Probably doesn't matter but, why is the radio button in a <th> and the "Case not listed" in a <td> ?  Shouldn't both be a <td> ?